iperf.py

```diff
@@ -122,14 +126,36 @@ def stop(self):
         r = self.exec_thread.join()
         if r.returncode != 0:
             logger.info(r)
-
         data = json.loads(r.out)
+        if self.test_type == TestType.IPERF_TCP:
+            self.print_tcp_results(data)
+        if self.test_type == TestType.IPERF_UDP:
+            self.print_udp_results(data)
+    
+    def print_tcp_results(self, data: dict):
```

Comment:
I guess the TCP results needs work as well to display as much info as UDP results.

Reply:
Yes, I can add this in a future PR